.github/workflows/pr-powershell.yml

```diff
@@ -1,34 +1,18 @@
 ---
 name: pr-powershell
-on:  # yamllint disable-line rule:truthy
+on: # yamllint disable-line rule:truthy
   pull_request:
     branches: [main]
     paths:
-      - '.github/workflows/pr-powershell.yml'
-      - '**.psd1'
-      - '**.psm1'
-      - '**.ps1'
+      - ".github/workflows/pr-powershell.yml"
+      - "**.psd1"
+      - "**.psm1"
+      - "**.ps1"
 
 env:
   LINTER_CONFIG_DIR: ./.github/config/linter
 
 jobs:
-  lint-powershell:
-    name: Lint PowerShell Files
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v3
-      - name: Install PSScriptAnalyzer module
-        shell: pwsh
-        run: |
-          Set-PSRepository PSGallery -InstallationPolicy Trusted
-          Install-Module PSScriptAnalyzer -ErrorAction Stop
-      - name: Lint PowerShell Files
-        shell: pwsh
-        run: |
-          Invoke-ScriptAnalyzer -Path . -Recurse -EnableExit `
```

Comment:
Curious: does mega linter have the script analyzer built in?

Reply:
Yes! If you look at the comment earlier in this PR, it will list out the linters that it runs and links for further information.